### PR TITLE
Ung 1559/process update upps

### PR DESCRIPTION
**Pull request by lalottacontinua**
the filter is supposed to react differently depending on the type of the incoming UPP. Here is a description of the different reactions: 
https://ubirch.atlassian.net/wiki/spaces/OPS/pages/1748074582/Filter+Service

### CODE REVIEW

File: src/main/scala/com/ubirch/filter/model/cache/CacheImpl.scala [hunk 1]
```diff
@@ -6,68 +6,57 @@ import com.ubirch.filter.ConfPaths.RedisConfPaths._
...
-  }
-
-  private def stopConnecting(): Unit = {
-    c.cancel()
-  }
+  /**
+    * This redis instance doesn't connect yet. On the first access
+    * a connection will be tried to become established. After that
+    * the instance keeps the connection open until a certain timeout
+    * is reached. (Haven't found the exact number yet.) For further
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/filter/model/cache/CacheImpl.scala:33]: good comment.
I think the default connection timeout is 2 seconds according to the link
```
connect-timeout = 2 seconds
```

Comment (lalottacontinua) [reply src/main/scala/com/ubirch/filter/model/cache/CacheImpl.scala:33]: thanks. :) 
I thought, that configuration is more about establishing the connection. As it's header is:  "# Maximum amount of time allowed to establish a connection to the Redis server". But I'm not sure. Will research a bit more later..

File: src/test/scala/com/ubirch/filter/services/kafka/FilterServiceIntegrationTestWithoutCache.scala [hunk 2]
```diff
@@ -117,62 +115,64 @@ class FilterServiceIntegrationTestWithoutCache extends TestBase with EmbeddedRed
...
+    "not change default behaviour when nothing is found in cassandra" in {
       implicit val kafkaConfig: EmbeddedKafkaConfig =
         EmbeddedKafkaConfig(kafkaPort = PortGiver.giveMeKafkaPort, zooKeeperPort = PortGiver.giveMeZookeeperPort)
       val bootstrapServers = "localhost:" + kafkaConfig.kafkaPort
 
       withRunningKafka {
 
         try {
-          "fuser -k 6379/tcp" !
+          redis.stop()
```
Comment (Hiroki6) [inline src/test/scala/com/ubirch/filter/services/kafka/FilterServiceIntegrationTestWithoutCache.scala:152]: Is there any reason that this `redis` object is defined globally?
I don't see the benefit. just make it difficult to see the current state of this object.
I think a RedisServer object should be defined in each test case.

Comment (Hiroki6) [reply src/test/scala/com/ubirch/filter/services/kafka/FilterServiceIntegrationTestWithoutCache.scala:152]: Ah, it may be because of `EmbeddedRedis` ?

**Pull request merged**